src/main/java/com/avispl/symphony/dal/infrastructure/management/qsc/qsyscore/QSYSCoreAggregatorCommunicator.java

```diff
@@ -1026,13 +1033,9 @@ private void retrieveTokenFromCore() {
 				}
 			}
 		} catch (CommandFailureException e) {
-			if (e.getStatusCode() == 403) {
-				this.loginInfo.setToken(null);
-			} else {
-				throw new ResourceNotReachableException("Error when login. Please check the credentials", e);
-			}
+			throw new ResourceNotReachableException("Error when login. Please check the credentials", e);
 		} catch (Exception e) {
-			throw new ResourceNotReachableException("Error when retrieve token", e);
+			throw new ResourceNotReachableException("Error when retrieve token. Please check the login request", e);
```

Comment:
Also here: "Unable to retrieve authorization token, login request failed." 
"Please check the login request" may be a bit confusing as a call-to-action :)